.github/workflows/pipeline.yaml

```diff
@@ -35,12 +38,25 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_HUB_ACCESS_USER }}
           password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+      - name: Login to azure
+        uses: azure/login@v1
+        with:
+          creds: '${{ secrets.AZURE_CREDENTIALS }}'
```

Comment:
This seems to work nicely and is cleaner than my usual approach of setting lots of env variables! 👍